packages/next/src/lib/picocolors.ts

```diff
@@ -47,35 +47,32 @@ const formatter =
       : open + string + close
   }
 
+const color = (...args: Parameters<typeof formatter>) =>
+  enabled ? formatter(...args) : String
```

Comment:
Could we avoid a new function and instead move the check for `enabled` boolean into `formatter`?

Such as:

```
const formatter = {
  if (!enabled) return String
  ...
}
```